package.json

```diff
@@ -120,8 +120,8 @@
     "azure-storage-legacy": "0.10.2",
     "mime": "~1.2.4",
     "moment": "~2.18.1",
-    "ms-rest": "^2.2.3",
-    "ms-rest-azure": "^2.4.1",
+    "ms-rest": "^2.2.5",
+    "ms-rest-azure": "^2.4.4",
```

Comment:
Can you change the line "test" in "scripts" . Basically move the test for ms-rest-azure earlier. This will enforce ms-rest to be present inside runtime/ms-rest-azure/node_modules and the CI should pass.

`
"test": "npm -s run-script jshint && npm -s run-script unit-msrestazure && npm -s run-script unit-arm && npm -s run-script unit && npm -s run-script unit-msrest",
`